appium/webdriver/extensions/android/nativekey.py

```diff
@@ -0,0 +1,1057 @@
+    BUTTON_16 = 203
+
+    # Key code constant: Language Switch key.
+    # Toggles the current input language such as switching between English and Japanese on
+    # a QWERTY keyboard.  On some devices, the same function may be performed by
+    # pressing Shift+Spacebar.
+    LANGUAGE_SWITCH = 204
+
+    # Key code constant: Manner Mode key.
+    # Toggles silent or vibrate mode on and off to make the device behave more politely
+    # in certain settings such as on a crowded train.  On some devices, the key may only
+    # operate when long-pressed.
+    MANNER_MODE = 205
+
+    # Key code constant: 3D Mode key.
+    # Toggles the display between 2D and 3D mode.
+    MODE_3D = 206
+
+    # Key code constant: Contacts special function key.
+    # Used to launch an address book application.
+    CONTACTS = 207
+
+    # Key code constant: Calendar special function key.
+    # Used to launch a calendar application.
+    CALENDAR = 208
+
+    # Key code constant: Music special function key.
+    # Used to launch a music player application.
+    MUSIC = 209
+
+    # Key code constant: Calculator special function key.
+    # Used to launch a calculator application.
+    CALCULATOR = 210
+
+    # Key code constant: Japanese full-width / half-width key.
+    ZENKAKU_HANKAKU = 211
+
+    # Key code constant: Japanese alphanumeric key.
+    EISU = 212
+
+    # Key code constant: Japanese non-conversion key.
+    MUHENKAN = 213
+
+    # Key code constant: Japanese conversion key.
+    HENKAN = 214
+
+    # Key code constant: Japanese katakana / hiragana key.
+    KATAKANA_HIRAGANA = 215
+
+    # Key code constant: Japanese Yen key.
+    YEN = 216
+
+    # Key code constant: Japanese Ro key.
+    RO = 217
+
+    # Key code constant: Japanese kana key.
+    KANA = 218
+
+    # Key code constant: Assist key.
+    # Launches the global assist activity.  Not delivered to applications.
+    ASSIST = 219
+
+    # Key code constant: Brightness Down key.
+    # Adjusts the screen brightness down.
+    BRIGHTNESS_DOWN = 220
+
+    # Key code constant: Brightness Up key.
+    # Adjusts the screen brightness up.
+    BRIGHTNESS_UP = 221
+
+    # Key code constant: Audio Track key.
+    # Switches the audio tracks.
+    MEDIA_AUDIO_TRACK = 222
+
+    # Key code constant: Sleep key.
+    # Puts the device to sleep.  Behaves somewhat like {@link #POWER} but it
+    # has no effect if the device is already asleep.
+    SLEEP = 223
+
+    # Key code constant: Wakeup key.
+    # Wakes up the device.  Behaves somewhat like {@link #POWER} but it
+    # has no effect if the device is already awake.
+    WAKEUP = 224
+
+    # Key code constant: Pairing key.
+    # Initiates peripheral pairing mode. Useful for pairing remote control
+    # devices or game controllers, especially if no other input mode is
+    # available.
+    PAIRING = 225
+
+    # Key code constant: Media Top Menu key.
+    # Goes to the top of media menu.
+    _MEDIA_TOP_MENU = 226
```

Comment:
`MEDIA_TOP_MENU` ?

Reply:
I don't know what precisely this constant for, I just ported from [this java code ](https://github.com/appium/java-client/blob/53814cd5ac77cf26e73c89eee49649af39f9c5a5/src/main/java/io/appium/java_client/android/nativekey/AndroidKey.java#L1024).

Reply:
I've checked with another member, it's just typo.
So please update to `MEDIA_TOP_MENU`.

Reply:
Do you mean to remove underscore in the beginning?